plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -16,8 +16,8 @@ import (
 	"sync"
 	"time"
 
-	dialout "github.com/cisco-ie/nx-telemetry-proto/mdt_dialout"
-	telemetry "github.com/cisco-ie/nx-telemetry-proto/telemetry_bis"
+	mdtdialout "github.com/cisco-ie/nx-telemetry-proto/mdt_dialout"
+	"github.com/cisco-ie/nx-telemetry-proto/telemetry_bis"
```

Comment:
Path to this package ends with `mdt_dialout` but name of the package is `mdtdialout`. It is easier for user to use name of the package.

Reply:
Agree for `mdtdialout`. Not sure about `telemetry` vs `telemetry_bis`.

Reply:
`telemetry_bis` is the original package name.

Reply:
But dus it mean anything? To me it looks like this could als have been named `github.com/cisco-ie/nx-telemetry-proto/telemetry_v2` or `github.com/cisco-ie/nx-telemetry-proto/telemetry/v2`. I still think it should be aliased as `telemetry`, but I let it open for others to give their opinion..

Reply:
Fine, changed to `telemetry`